test/ssc_test/cmod_battery_stateful_test.cpp

```diff
@@ -95,3 +95,17 @@ TEST_F(CMBatteryStatefulIntegration_cmod_battery_stateful, SubMinute) {
     EXPECT_NEAR(V, 492.50, 1e-2);
     EXPECT_NEAR(SOC, 52.05, 1e-2);
 }
+
+TEST_F(CMBatteryStatefulIntegration_cmod_battery_stateful, ReadJson) {
+    CreateModel(1./60.);
+
+    auto copy = json_to_ssc_data(js.c_str());
+
+    mod = ssc_stateful_module_create("battery_stateful", copy);
+    copy = json_to_ssc_data(js.c_str());
+    ssc_module_exec(mod, copy);
+
+    js = ssc_data_to_json(copy);
+
+    printf("%s\n", js.c_str());
```

Comment:
Does this test need to assert/expect something?

Reply:
Fixed